internal/e2e/rollup_code_splitting/BUILD.bazel

```diff
@@ -7,7 +7,19 @@ rollup_bundle(
         ["*.js"],
         exclude = ["*.spec.js"],
     ),
+    enable_code_splitting = True,
+    entry_point = ":main1.js",
+    license_banner = ":license.txt",
+)
+
+rollup_bundle(
+    name = "bundle_multi_entry",
+    srcs = glob(
+        ["*.js"],
+        exclude = ["*.spec.js"],
+    ),
     additional_entry_points = ["build_bazel_rules_nodejs/internal/e2e/rollup_code_splitting/additional_entry.js"],
+    enable_code_splitting = False,
```

Comment:
This would implied to be true because `additional_entry_points` is specified?

Reply:
yeah this should have been an error, will fix